src/include/processor/operator/recursive_extend/frontier_scanner.h

```diff
@@ -64,9 +64,11 @@ class PathScanner : public BaseFrontierScanner {
 public:
     PathScanner(TargetDstNodes* targetDstNodes, size_t k,
         std::unordered_map<common::table_id_t, std::string> tableIDToName,
-        path_semantic_check_t semanticCheckFunc, common::ExtendDirection extendDirection)
+        path_semantic_check_t semanticCheckFunc, common::ExtendDirection extendDirection,
```

Comment:
didn't notice this. but isn't `path_semantic_check_func` easier to understand than `path_semantic_check_t`?